### ISSUE TITLE
tflite and tflite-runtime library

### ISSUE THREAD

**Issue by WikiGenius** [Deci-AI/super-gradients#937]
Does the model support TFLite and TFLite runtime?

I want to run the model on the mobile. so I am asking if this is available 

Thanks so much

**Comment by NatanBagrov** [comment]
Hello @WikiGenius, this is not supported out of the box.

You can use, for example, [onnx2keras repo](https://github.com/gmalivenko/onnx2keras) to convert an ONNX (which you can export using SG) to Keras, then compile to TFLite using the official documentation of TensorFlow.

**Comment by WikiGenius** [comment]
So, onnex2tf then tensor flow2tflite
No problem. But I hope there are no problems in this conversation like Yolo V8.

Also, I hope tflite run time library can handle tflite of this model.

**Comment by NatanBagrov** [comment]
Good luck! Closing the thread

**Comment by WikiGenius** [comment]
I tested the performance of TFLite of YOLO NAS on the PC. It is very slow and not relevant to embedded devices. I think YOLOv8n is better suited for embedded devices for several reasons. Firstly, the output has 8400 points. Secondly, the number of parameters is huge.

This experiment on the PC used a pre-trained model of YOLO NAS, focusing only on persons without NMS.

![video_3](https://github.com/Deci-AI/super-gradients/assets/33193541/44a54cc8-8eca-4e49-8050-d73365fa7262)

**Comment by WikiGenius** [comment]
I tested the performance of TFLite YOLO NAS on the PC, and it was found to be very slow, making it unsuitable for embedded devices. In my opinion, YOLOv8n is better suited for embedded devices due to several reasons. Firstly,  NAS provides an output with 8400 points. Secondly, it has a huge number of parameters.

In this experiment on the PC, a pre-trained model of YOLO NAS was used, specifically focusing on
![video_3](https://github.com/Deci-AI/super-gradients/assets/33193541/4d399cf9-a2b4-4025-84f8-a06a5d7f8902)
 persons without non-maximum suppression (NMS), and it utilized multi-threading techniques.
